pkg/controller/certapi/certificate.go

```diff
@@ -152,8 +156,13 @@ func (ca *CertificateAPI) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	// Create the Certificate
 	now := time.Now().UTC()
 	claims := verifyapi.NewVerificationClaims()
-	// This server curently does not provide any transmission risk overrides, although that is part
-	// of the diagnosis verification protocol.
+	// Assign the report type.
+	claims.ReportType = subject.TestType
+	if subject.SymptomDate != nil {
+		claims.SymptomOnsetInterval = subject.SymptomInterval()
+	}
+
+	// TODO(mikehelmick): Assign transmission risk overrides. Algorithm not set yet.
```

Comment:
is this planned soon, or should we make a tracking issue?

Reply:
actively working on it - part of this same issue.